src/pyclaw/sharpclaw/ClawParams.f90

```diff
@@ -9,7 +9,7 @@ module ClawParams
     integer :: num_dim, num_waves, index_capa
 
     ! Method-related parameters:
-    integer :: char_decomp, lim_type, weno_order
+    integer :: char_decomp, lim_type, reconstruction_order
```

Comment:
Maybe that's a good idea; we can just emit a warning saying that the old attribute will be deprecated in future versions.